README.md

```diff
@@ -98,3 +98,23 @@ ios_application(
 
 See the [examples](https://github.com/bazelbuild/rules_apple/tree/master/examples)
 directory for sample applications.
+
+## Supported bazel versions
+
+rules_apple and rules_swift are often affected by changes in bazel
+itself. This means you generally need to update these rules as you
+update bazel.
+
+You can also see the supported bazel versions in the notes for each
+release on the [releases
+page](https://github.com/bazelbuild/rules_apple/releases).
+
+Besides these constraint this repo follows [semver](https://semver.org/)
+as best as we can since the 1.0.0 release.
+
+| Bazel release | Minimum supported rules version | Final supported rules version|
+|:-------------------:|:-------------------:|:-------------------------:|
+| 6.x (most recent rolling) | 0.34.0 | current |
+| 5.x | 0.33.0 | current |
```

Comment:
Technically did we support 5.x before 0.33.0? Or did we need to make an update just to support it? Seems like there would be some overlap in these version ranges.

Reply:
yea these aren't perfect, this was the first version where we mentioned the release was tested with it, but it _might_ have been supported sooner, hard to say without testing. we can fix these up if folks find they're inaccurate and need to show that